opensearch.tf

```diff
@@ -0,0 +1,49 @@
+locals {
+  cluster_name   = "opensearch-${terraform.workspace}"
+  cluster_domain = "aepps.com"
+}
+
+data "aws_region" "current" {}
+
+provider "elasticsearch" {
+  url                   = "https://${local.cluster_name}.${local.cluster_domain}"
+  aws_region            = data.aws_region.current.name
+  elasticsearch_version = "7.10.2"
```

Comment:
it's pretty old, are there any reason to not use newest?

Reply:
> about `e es admin user password is provided with main.auto.tfvars file , in superhero tf code I used git-secret. If someone doesn't have another proposal , will use it here.` @dincho made awesomnes with vault + secrets etc,, i would proceed with this kind of a secrets to have common ground everywhere, if we would like to use something else there is aws secret manager as far as i rmember?

I think they are for different use cases , if we use some of them , we still have to provide some secret to terraform. Correct if I'm wrong and we can configure what you have in mind.

Reply:
> it's pretty old, are there any reason to not use newest?

Will check again the opensearch versions and will fix.They have and auto update option for opensearch domain.

Reply:
I answer you in another thread above.